### DISCUSSION TITLE
[Simple Feature/WebUI] Adding a button to hide the sidebar

### DISCUSSION THREAD

**Discussion by samyxdev** [langfuse/langfuse#1118] [Ideas]
### Describe the feature or potential improvement

Can a button be added to hide the sidebar? it would make browsing easier to use when many columns are enabled (for e.g. on Traces).

### Contribute

- [ ] Yes, I can implement this and raise a PR

**Comment by marcklingen** [comment]
Thanks for the suggestion @samyxdev. Agree, this would be super nice as especially the single trace view would benefit a lot from more horizontal space on large screens.
